app/modules/Application/Mvc/Helper/LangSwitcher.php

```diff
@@ -70,4 +73,28 @@ private function changeRouteName($route_name, $lang)
         return $route_name . '_' . $lang;
     }
 
-} 
+    private function getMLSlug($matched_route, $route_params, $lang)
```

Comment:
Also note: the `getMLSlug` will be used in public `render`. So once you made it private the `render` can't be extended in child class.